script.js

```diff
@@ -0,0 +1,132 @@
+API_KEY = "a7985326430d6fbc3cd513d95d89bb20"
+let url = "https://api.themoviedb.org/3/movie/now_playing?api_key=a7985326430d6fbc3cd513d95d89bb20"
+let searchURL = "https://api.themoviedb.org/3/search/movie?query="
+
+ //create star
+ function generateCards(movieObject) {
+    let star = document.createElement('span');
+    star.classList.add('star');
+    let starContent = document.createTextNode('⭐️');
+    star.appendChild(starContent);
+
+//create rating 
+    let rating = document.createElement('span');
+    rating.classList.add('movie-votes');
+    let ratingContent = document.createTextNode(movieObject.vote_average);
+    rating.appendChild(ratingContent);
+   
```

Comment:
recommend using helper functions to make code more DRY (e.g. creating and appending elements)